.github/workflows/build.yml

```diff
@@ -115,7 +115,7 @@ jobs:
             type=semver,pattern={{version}}
             type=semver,pattern={{major}}
             type=semver,pattern={{major}}.{{minor}}
-            type=raw,value=1.9.5-SNAPSHOT,enable=${{ github.ref == format('refs/heads/{0}', 'main') }}
+            type=raw,value=1.9.8,enable=${{ github.ref == format('refs/heads/{0}', 'main') }}
```

Comment:
this should not be necessary. If the build is triggered by a tag like `v1.2.3` for example, the rules above automatically create tags `:1`, `:1.2` and `:1.2.3`. 
The `type=ref,event=branch` already would create and overwrite `main`, in case the latest release should be used somewhere.
In general the declaration is fine though. I just think it's a lot of manual effort to always bump this by hand